src/mn_setup.c

```diff
@@ -3080,7 +3080,11 @@ static const char *gyro_action_strings[] = {
     "None",
     "Disable Gyro",
     "Enable Gyro",
-    "Invert"
+    "Invert",
+    "Reset Camera",
+    "Reset / Disable Gyro",
```

Comment:
Maybe "Disable Gyro / Reset"? I think the button's main purpose is still to enable Gyro.

Reply:
"Primary Function (Tap) / Secondary Function (Hold)" is what I've seen in other games. Unfortunately, there's no room to be that descriptive here, so maybe there's some confusion?

Reply:
Maybe it's just that I never play "other games". 😆